wolfssl/wolfcrypt/types.h

```diff
@@ -725,7 +725,11 @@ enum {
 
 #elif defined(WOLFSSL_LINUXKM)
 
-    /* definitions are in linuxkm/linuxkm_wc_port.h */
+    /* definitions are in linuxkm/linuxkm_wc_port.h. */
+
+#elif defined(WOLFSSL_BSDKM)
+
+    /* definitions are in bsdkm/bsdkm_wc_port.h */
```

Comment:
You added a period to the linuxkm comment, then left it off the bsdkm comment 😜️